<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consoleWgt</class>
 <widget class="QWidget" name="consoleWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#consoleWgt{
background:#1f2020;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
background:#1f2020;
box-shadow:0 10px 20px 3px rgba(0,0,0,0.50);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="commonPageWgt" name="playPage_" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="commonPageWgt" name="replayPage_" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="commonPageWgt" name="devicePage_" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="commonPageWgt" name="storagePage_" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="commonPageWgt" name="VRMapPage" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="commonPageWgt" name="VRViewPage" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="commonPageWgt" name="IOTPage_" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>325</width>
            <height>126</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>commonPageWgt</class>
   <extends>QWidget</extends>
   <header>commonpagewgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
